### ISSUE TITLE
PEP 3105 Make print a function in twisted/scripts

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8399]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8399|
|Type|enhancement|
|Created|2016-06-03 10:47:43Z|

This ticket was split out of [ticket:5812] which was too big to review.

PEP 3105 ( ​​​https://www.python.org/dev/peps/pep-3105/ ) **print** deprecates has changed from a statement to a function.

In Python 2.6 and Python 2.7, the default is print as a statement

```
print "Hello"
```

In Python 3.x, the above example is an error, since only print as a function is available.

To consistently use **print** as a function
on Python 2.6, Python 2.7, and Python 3.x, this should be done:

```
from __future__ import print_function

print("Hello")
```

<details><summary>Searchable metadata</summary>

```
trac-id__8399 8399
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1464950863627877 1464950863627877
changetime__1465027924017082 1465027924017082
version__None None
owner__Ying_Li__cyli_____ Ying Li <cyli@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/115

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli set owner to @cyli<br>@cyli set status to `closed`|
|-|-|

In changeset 8ebcec5473d6c716ff3278c72f6a17b6b90f64ea
```
#!CommitTicketReference repository="" revision="8ebcec5473d6c716ff3278c72f6a17b6b90f64ea"
Merge pull request #115 from rodrigc/8399-rodrigc-print: Convert from print statement to print() function in twisted/scripts

Author: rodrigc
Reviewer: cyli
Fixes: #8399

Convert from print statement to print() function in twisted/scripts
```
